anydex/test/wallet/stellar/test_xlm_db.py

```diff
@@ -17,11 +17,14 @@ def setUp(self):
                               source_account='GDQWI6FKB72DPOJE4CGYCFQZKRPQQIOYXRMZ5KEVGXMG6UUTGJMBCASH',
                               operation_count=1,
                               transaction_envelope="AAAAAOFkeKoP9De5JOCNgRYZVF8IIdi8WZ6olTXYb1KTMlgRAAAAZAGOO"
-                                                   "+wAAAA9AAAAAQAAAAAAAAAAAAAAAAAAAAAAAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-                                                   "AAAAAAAAAAAAAAABAAAAAQAAAADhZHiqD/Q3uSTgjYEWGVRfCCHYvFmeqJU12G9SkzJYEQA"
-                                                   "AAAAAAAAAXQbflbQZVvhfaHtF6ESvgGrNnl2gi44084MWUaGbmNkAAAAAAcnDgAAAAAAAAA"
-                                                   "ABHvBc2AAAAEBUL1wo8IGHEpgpQ7llGaFE+rC9v5kk2KPJe53/gIdWF+792HYg5yTTmhJII"
-                                                   "97YgM+Be8yponPH0YjMjeYphewI",
+                                                   "+wAAAA9AAAAAQAAAAAAAAAAAAAAAAAAAAAAAA"
+                                                   "ADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                                                   "AAAAAAAAAAAAAAABAAAAAQAAAADhZHiqD/Q3uSTgjYEWGVRfC"
+                                                   "CHYvFmeqJU12G9SkzJYEQA"
+                                                   "AAAAAAAAAXQbflbQZVvhfaHtF6ESvgGrNnl2gi44084MWUaGbmNk"
+                                                   "AAAAAAcnDgAAAAAAAAA"
+                                                   "ABHvBc2AAAAEBUL1wo8IGHEpgpQ7llGaFE+rC9v5kk2KP"
+                                                   "Je53/gIdWF+792HYg5yTTmhJII97YgM+Be8yponPH0YjMjeYphewI",
```

Comment:
Truly one of the finest pieces of reformatting I have ever seen 😂

Reply:
xd